.github/workflows/release_render.yml

```diff
@@ -0,0 +1,71 @@
+name: Create Release for Render
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version to release, e.g. v0.1.0-render'
+        required: true
+        default: 'v0.1.0-render'
```

Comment:
@moolen My only concern here is that if it sets this as latest version people might think that's the version for ESO. But it's just a render version. The reason it's not render/v0.1.0 is that goreleaser fails on that complaining that it's not valid semver... Any suggestions?

Reply:
This is fine, all other alternatives are worse IMHO. Baking it into the ESO release process would also be an option, though that would make the build process much more convoluted. 
I think it's fine to start with this. keep it simple.

Reply:
Alright! Agreed. 👍 thanks ☺️